android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -246,6 +248,40 @@ class Screen(context: ReactContext?) : FabricEnabledViewGroup(context) {
 
     var nativeBackButtonDismissalEnabled: Boolean = true
 
+    fun startRemovalTransition() {
+        if (!isBeingRemoved) {
+            isBeingRemoved = true
+            startTransitionRecursive(this)
+        }
```

Comment:
Can we have it called multiple times? Why do we have the multiple-entry guard here?